### PR TITLE
latex & bibtex: strip mathml

### PR DESCRIPTION
**Pull request by MJedr**
ref: cern-sis/issues-inspire#147

### CODE REVIEW

File: backend/inspirehep/records/marshmallow/literature/bibtex.py [hunk 1]
```diff
@@ -234,7 +235,8 @@ def get_title(self, data):
         if "subtitle" in title_dict:
             title_parts.append(title_dict["subtitle"])
         return ": ".join(
-            f"{{{latex_encode(part, contains_math=True)}}}" for part in title_parts
+            f"{{{latex_encode(remove_tags(remove_tags(part, allowed_tags='DUMMYROOTTAG')), contains_math=True)}}}"
```
Comment (drjova) [inline backend/inspirehep/records/marshmallow/literature/bibtex.py:238]: can we do it in a loop? it's getting a hard to read having everything in the formatted string

File: backend/inspirehep/records/marshmallow/literature/bibtex.py [hunk 2]
```diff
@@ -233,9 +234,17 @@ def get_title(self, data):
...
-        )
+
+        formatted_parts = []
+        for part in title_parts:
+            part_witouth_mathml = remove_tags(
+                remove_tags(part, allowed_tags="DUMMYROOTTAG")
+            )
+            latex_encoded_part = (
+                f"{{{latex_encode(part_witouth_mathml, contains_math=True)}}}"
+            )
```
Comment (drjova) [inline backend/inspirehep/records/marshmallow/literature/bibtex.py:245]: hmm I don't understand ` f"{{{ }}}"` need

Comment (MJedr) [reply backend/inspirehep/records/marshmallow/literature/bibtex.py:245]: me neither, but it was like that before and I don't know latex and bibtex at all so I was afraid to remove it 😅 I can remove it

Comment (drjova) [reply backend/inspirehep/records/marshmallow/literature/bibtex.py:245]: Let’s try to remove it and see :)

Comment (MJedr) [reply backend/inspirehep/records/marshmallow/literature/bibtex.py]: doesn't work, it's about adding `{}` to the encoded text, I'll leave it as it was

**Review by drjova — APPROVED** [review]

**Pull request merged**